Libraries/LibWeb/SVG/SVGScriptElement.cpp

```diff
@@ -63,10 +126,7 @@ void SVGScriptElement::process_the_script_element()
     if (!m_document->ready_to_run_scripts())
         HTML::main_thread_event_loop().spin_until(GC::create_function(heap(), [&] { return m_document->ready_to_run_scripts(); }));
 
```

Comment:
does this happen? 

> Note that a [load](https://www.w3.org/TR/SVGMobile12/interact.html#EventsLoad) event is dispatched on a ['script'](https://www.w3.org/TR/SVGMobile12/script.html#ScriptElement) element once it has been processed, unless it referenced external script content with an [invalid IRI reference](https://www.w3.org/TR/SVGMobile12/intro.html#TermInvalidIRIReference) and ['externalResourcesRequired'](https://www.w3.org/TR/SVGMobile12/struct.html#ExternalResourcesRequiredAttribute) was set to 'true'.

(from the SVGMobile12 spec)

If not a FIXME might be in order.

Reply:
This does not happen. Adding a fixme